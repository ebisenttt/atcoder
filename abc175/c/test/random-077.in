668451156974198 754230342633427 699503202106429
