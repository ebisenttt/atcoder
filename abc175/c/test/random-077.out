31052045132231
